### PR TITLE
Fix make

### PR DESCRIPTION
**Pull request by tevino**
https://github.com/src-d/go-license-detector/issues/60

Should I add the above link into the commit message?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ The intended area of it's usage is data mining.
 ```
 export GO111MODULE=on
 go mod download
+make
```
Comment (vmarkovtsev) [inline README.md]: The bindata is already generated, this should not be required. `make` generates it if it does not exist.

File: .travis.yml [hunk 2]
```diff
@@ -6,17 +6,19 @@ go_import_path: gopkg.in/src-d/go-license-detector.v2
...
 
 before_install:
   - go get -v golang.org/x/lint/golint
   - go get -v github.com/haya14busa/goverage
+  - go get -u -v github.com/jteeuwen/go-bindata/...
 
 install:
   - export GO111MODULE=on
-  - go mod vendor
+  - make
```
Comment (vmarkovtsev) [inline .travis.yml]:
```suggestion
  - go mod download
```

File: .travis.yml [hunk 3]
```diff
@@ -6,17 +6,19 @@ go_import_path: gopkg.in/src-d/go-license-detector.v2
...
   - 1.12.x
+  - 1.13.x
 
 matrix:
   fast_finish: true
 
 before_install:
   - go get -v golang.org/x/lint/golint
   - go get -v github.com/haya14busa/goverage
+  - go get -u -v github.com/jteeuwen/go-bindata/...
```
Comment (vmarkovtsev) [inline .travis.yml]: Hence this is not required

File: README.md (hunk 1 above)
Comment (tevino) [reply README.md]: I found that the CI complains 

`flag provided but not defined: -nometadata` 
https://travis-ci.com/src-d/go-license-detector/jobs/233963633

So I thought an older version of `bindata` was used, but it seems not.

Comment (vmarkovtsev) [reply README.md]: OK so let's re-generate bindata in the CI so that we test `make`, but remove it from the README.

**Comment by vmarkovtsev** [comment]
Interesting 1.13 build passed, but 1.11 and 1.12 did not: they do not like the absence of go.mod in jteeuwen/go-bindata

**Comment by tevino** [comment]
> Interesting 1.13 build passed, but 1.11 and 1.12 did not: they do not like the absence of go.mod in jteeuwen/go-bindata

Indeed, I haven't figure out a good way to install the latest `go-bindata` that works on v1.11 ~ v1.13.

Do you have any idea?

File: README.md (hunk 1 above)
Comment (tevino) [reply README.md]: The PR started getting messy, I squashed some commits.

I noted the detail about old version of `go-bindata` [here](https://github.com/src-d/go-license-detector/pull/61/commits/84eefa098e178c29934258b767412eebafa48563)

**Comment by tevino** [comment]
Works now!

**Comment by vmarkovtsev** [comment]
Thank you. I will make a new release in a few hours.

**Comment by vmarkovtsev** [comment]
v3 has been released

**Pull request merged**